lib/DDG/Spice/Cargo.pm

```diff
@@ -0,0 +1,27 @@
+package DDG::Spice::Cargo;
+# ABSTRACT: Returns package information from the cargo package manager's registry.
+
+use strict;
+use DDG::Spice;
+
+primary_example_queries "cargo gcc";
+description "Shows a Cargo package";
+name "Cargo";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Cargo.pm";
+icon_url "/i/crates.io.ico";
+topics "sysadmin", "programming";
+category "programming";
+attribution github  => ['https://github.com/tombebbington', 'Tom Bebbington'],
+            twitter => ['https://twitter.com/tomdbebbington', '@Tom Bebbington'];
+
+triggers startend => 'cargo';
```

Comment:
"rust cargo" could be a trigger as well. The Sublime Packages IA uses "sublime packages" as th example.

NPM and RubyGems use their names as the triggers, but those are much less likely to signal an unrelated query.